### DISCUSSION TITLE
How can I get the value of sessionId from odSpDocumentDeltaCOnnection joinSession call?

### DISCUSSION THREAD

**Discussion by jindalharshit** [microsoft/FluidFramework#14271] [Q&A]
### Problem
I need access to session Id to perform some operation every time a new session is created.

### Investigation

[Here ](https://github.com/microsoft/FluidFramework/blob/446b4ea9fcea8b761ec5de6800821299095afcc0/packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts#:~:text=const%20connection%20%3D%20await%20this,return%20connection%3B) the websocketEndpoint ID received as the result of joinSession call is being passed as documentId. 

This means that the documentId is same as the sessionId and I should be able to access it using the DeltaManager.ConnectionManager.ConnectionProps.DocumentId [object](https://github.com/microsoft/FluidFramework/blob/7eef70a4b3b94f9bdf2ba6ad8e42ed31394c404d/packages/loader/container-loader/src/deltaManager.ts)

Can someone confirm if this understanding is correct?

**Comment by vladsud** [comment]
@jindalharshit, could you please describe your scenario?
The terms you point to are internal implementation details, they are not exposed and could change in future, so I'm not really sure if (and why) they might be relevant to your scenario.
